presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemorySmoke.java

```diff
@@ -185,6 +185,31 @@ public void testViews()
         assertQueryFails("DROP VIEW test_view", "line 1:1: View 'memory.default.test_view' does not exist");
     }
 
+    @Test
+    public void testRenameView()
```

Comment:
Wondering if we should instead add these tests to `AbstractTestDistributedQueries`? This way all connectors which support views can run this and we don't need to duplicate code.

Reply:
@ZacBlanco - The support for ALTER VIEW RENAME TO is implemented only for the Iceberg and Memory connectors. If we add a test for this in AbstractTestDistributedQueries, it is required to override it in each connector, so the test will not fail for the other connectors.